core/src/stories/Migration/Migration.stories.mdx

```diff
@@ -1,756 +1,783 @@
+  size="lg"
+  state="none"
+  variant="default"
+  label="Label text"
+  label-position="outside"
+  placeholder="Placeholder"
+  ></tds-text-field>
+```
+
+#### Changed prop name 'nominwidth'
+
+The prop `nominwidth` was changed to `no-min-width` to conform with other prop names.
+
+##### What action is required?
+
+Rename all instances of `nominwidth` to `no-min-width`.
+
+```jsx
+// Old ❌
+<sdds-textfield
+    nominwidth
+></sdds-textfield>
+
+// New ✅
+<tds-text-field
+    no-min-width
+></tds-text-field>
+```
+
+#### Removed slot for helper and added `helper` prop
+
+Previously the textfield has had its helper text set via a slot. This is now changed
+to be a prop instead, to align with other components.
+
+##### What action is required?
+
+Change all instaces of `<span slot='sdds-helper'>${helper}</span>` to attrubute
+`helper="${helper}"`
+
+```jsx
+// Old ❌
+<sdds-textfield
+>
+  <span slot='sdds-helper'>Helper text!</span>
+</sdds-textfield>
+
+// New ✅
+<tds-text-field
+    helper="Helper text"
+></tds-text-field>
+```
+
+#### Added prop for mode-variant and removed class `sdds-on-white-bg` on container element
+
+Previously the class `sdds-on-white-bg` was used to determine the mode variant. We have now introduced the prop `mode-variant` with values `primary` or
+`secondary`. `primary` is the default value and represents the old `sdds-on-white-bg`
+
+##### What action is required?
+
+Replace all instances of `class="sdds-on-white-bg"` on container element with the prop `mode-variant` with `primary` as value directly on the component.
+If the component is used on a background with a color of grey-50, mode-variant prop value `secondary` should be passed.
+
+```jsx
+// Old ❌
+<div class="sdds-on-white-bg">
+  <sdds-textfield> </sdds-textfield>
+</div>
+// New ✅
+ <tds-text-field mode-variant="primary">
+ </tds-text-field>
+```
+
+## Table
+
+[Story](/docs/components-data-table--default)
+
+#### Renamed `enablePagination`, `enableMultiselect` and `enableFilter` props to `pagination`, `multiselect` and `filter`.
+
+##### What action is required?
+
+```jsx
+// Old ❌
+<sdds-table enable-multiselect>
+  <sdds-table-header enable-filter>
+    //
+  </sdds-table-header>
+  <sdds-table-body enable-dummy-data>
+    //
+  </sdds-table-body>
+  <sdds-table-footer enable-pagination></sdds-table-footer>
+</sdds-table>
+
+// New ✅
+<tds-table multiselect>
+  <tds-table-header filter>
+    //
+  </tds-table-header>
+  <tds-table-body>
+    //
+  </tds-table-body>
+  <tds-table-footer pagination ></tds-table-footer>
+</tds-table>
+```
+
+#### Removed `disableFilteringFunction`, `disableSortingFunction` and `enableClientPagination`.
+
+Previously, the data table component provided these props to enable users to implement their own filter, sort, and pagination logic. However, we have now removed these props and replaced them with cancellable events emitted by the component.
+
+The events are: 'tdsPageChange', 'tdsSortChange', 'tdsFilterChange'
```

Comment:
Are these event names still in use? Can't find them in the code base

Reply:
Martin did update them now.